### ISSUE TITLE
ENH: include error model plots in denoise-* output

### ISSUE THREAD

**Issue by nbokulich** [qiime2/q2-dada2#158]
**Improvement Description**
The error model plots output by dada2 can be useful for diagnosing issues with the model. This is used in tandem with the denoising-stats output to evaluate outcomes, e.g., too many reads lost at the denoising stage could indicate a poor error model. The plots output by q2-dada2 should be included in the outputs of `denoise-*` to assist with troubleshooting.

**Current Behavior**
No error model information is output.

**Proposed Behavior**
Include error model plots from `plotErrors()` as a separate tab in the denoise-stats output.

**Comment by gregcaporaso** [comment]
@jordenrabasco, any chance this would be something you'd be interested and available to work on? 

cc @benjjneb

**Comment by jordenrabasco** [comment]
@gregcaporaso Yeah I am definitely interested! I am currently out of town until the end of the month, but then I should have a few weeks relatively free in early Aug where I can look into this. Does that work for a timeline on your end?

**Comment by gregcaporaso** [comment]
Definitely, thanks for your interest @jordenrabasco!
